arc/runner/system-exp-aws/values.yaml

```diff
@@ -4,7 +4,7 @@ containerMode:
   type: kubernetes
   kubernetesModeWorkVolumeClaim:
     accessModes: [ReadWriteOnce]
-    storageClassName: ebs-gp3-encrypted
+    # storageClassName: ebs-gp3-encrypted
```

Comment:
Sorry if I'm being picky - why is this commented out?

Reply:
Not picky. I was testing, it's removed new (and moved to `common-values.yaml`)